modules/@apostrophecms/schema/index.js

```diff
@@ -1484,7 +1485,7 @@ module.exports = {
           ) {
             throw self.apos.error('invalid');
           }
-          choices = await self.apos.modules[field.moduleName][field.choices](req);
+          choices = await self.apos.modules[field.moduleName][field.choices](req, docId);
```

Comment:
Please put that in an object, I'm sure there will be more.

Reply:
i knew this!!!